README.md

```diff
@@ -10,6 +10,23 @@ If there is anything you are looking for that is not covered here you should ref
 
 ----
 
+### Statement Termination ###
+
+Unlike Objective-C, omit the use of `;` to terminate statements. Instead, simply use new lines to indicate the end of a statement. 
+
+```swift
+let myVar = 0
+doSomething(myVar)
+
+return
+```
+
+Avoid multiple statements on a single line.
+
+```swift
+guard let obj = myObj else { print("Something went wrong"); return; } // Wrong! Instead, place each item on its own line.
```

Comment:
are we sure about that? Sometimes I like putting everything on the same line if it's still easy to read, especially when you don't need to have a print

`guard let a = a else { return }`
Thoughts?